src/server/list_family.cc

```diff
@@ -338,7 +341,7 @@ OpResult<string> OpRPopLPushSingleShard(const OpArgs& op_args, string_view src,
 
 // Read-only peek operation that determines wether the list exists and optionally
 // returns the first from right value without popping it from the list.
```

Comment:
the comment should be changes (it's not first on the right anymore). also, `wether` is a typo, please fix on the way :)

Reply:
> the comment should be changes (it's not first on the right anymore). also, `wether` is a typo, please fix on the way :)

done